presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -18,7 +18,6 @@
 import com.facebook.presto.metadata.TableHandle;
 import com.facebook.presto.spi.ColumnHandle;
 import com.facebook.presto.spi.type.Type;
-import com.facebook.presto.sql.tree.DefaultTraversalVisitor;
 import com.facebook.presto.sql.tree.ExistsPredicate;
```

Comment:
Fix cmt msg

* "was't used actually used"
* "Additionally, it's doesn't" - use past simple?